sklearn/datasets/tests/test_openml.py

```diff
@@ -33,6 +32,7 @@
 OPENML_TEST_DATA_MODULE = "sklearn.datasets.tests.data.openml"
 # if True, urlopen will be monkey patched to only use local files
 test_offline = True
+_DATA_FILE = "data/v1/download/{}"
```

Comment:
it might be worth to either put a small comment or change the variable name because it might be a bit fuzzy what it means. In the previous PR are named it `_MONKEY_PATCH_LOCAL_OPENML_PATH"` which was probably too long. Also, since it is only define in the test, we might not need anymore the leading underscore.